homeassistant/components/plugwise/gateway.py

```diff
@@ -88,3 +98,16 @@ async def async_unload_entry_gw(hass: HomeAssistant, entry: ConfigEntry):
     ):
         hass.data[DOMAIN].pop(entry.entry_id)
     return unload_ok
+
+
+@callback
+def async_migrate_entity_entry(entry: RegistryEntry) -> dict[str, Any] | None:
+    """Migrate Plugwise entity entries.
+
+    - Migrates unique ID from old relay switches to the new unique ID
+    """
+    if entry.domain == Platform.SWITCH and entry.unique_id.endswith("-plug"):
```

Comment:
I'm not sure if we should use the Platform enum for the entity registry api. The `entry.domain` is not exclusive to platform domains. A Person entity will register entity entries under the `"person"` domain eg which is not a platform.

Reply:
👍

Reply:
Adjusted in [9e77f7b](https://github.com/home-assistant/core/pull/66174/commits/9e77f7b77ba5a794c5ade48e31d6709f69b3c41d)